src/setup/config.rs

```diff
@@ -72,7 +72,7 @@ impl NodeMetaData {
 
 /// Fields to be written to the node's configuration file.
 #[derive(Debug)]
-pub struct NodeConfig {
+pub struct NewNodeConfig {
```

Comment:
Why change the naming here?

Reply:
NewNodeConfig is used to setup a 'new' nodes (in contrary to nodes that read existing ledger data).